IsraelHiking.Web/src/application/services/spatial.service.ts

```diff
@@ -363,4 +363,9 @@ export class SpatialService {
             pixelY: Math.floor((tile.y - Math.floor(tile.y)) * tileSize)
         };
     }
+
+    public static isInIsrael(latlng: LatLngAlt): boolean {
+        return latlng.lat > 29 && latlng.lat < 34 &&
+                latlng.lng > 34 && latlng.lng < 36;
```

Comment:
```suggestion
        return latlng.lat > 29.37711 && latlng.lat < 33.35091 &&
                latlng.lng > 34.07929 && latlng.lng < 29.37711;
```

Use the [Geofabrik polygon](https://download.geofabrik.de/asia/israel-and-palestine.poly)'s bounds

Reply:
I don't think the last number is correct, right? `latlng.lng < 29.37711`

Reply:
I've edited the suggestion, please review it and let me know if it's correct.

Reply:
The raw numbers are:


  | Min | Max
-- | -- | --
Lat | 29.37711 | 33.35091
Lon | 34.07929 | 35.91531

I fixed the code to reflect that